services/dask-sidecar/requirements/_base.txt

```diff
@@ -6,8 +6,10 @@
 #
 aiodocker==0.21.0
     # via -r requirements/_base.in
-aiofiles==0.7.0
-    # via -r requirements/_base.in
+aiofiles==0.8.0
```

Comment:
why is this upgraded here?

Reply:
So I've added the aiofiles requirement on the service-library and upgraded the dependency on all projects which use it.

Reply:
so what about the catalog? why is it still 0.5?